src/parsers/pm_io/data.jl

```diff
@@ -611,7 +611,7 @@ function check_thermal_limits(data::Dict{String,Any})
     for branch in branches
         if !haskey(branch, "rate_a")
             if haskey(data, "conductors")
-                @error("Multiconductor Not Supported in PowerSystems")
+                error("Multiconductor Not Supported in PowerSystems")
                 #branch["rate_a"] = MultiConductorVector(0.0, data["conductors"])
```

Comment:
Is there a reason to keep the commented-out code around?

Reply:
Just in case we want it for reference later. But we can delete it too.